### PR TITLE
chore: update catalog addon deployment docs

### PR DESCRIPTION
**Pull request by natmegs**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.d2iq.com/browse/D2IQ-67163

## Description of changes being made

Adds details about how parameters are specified for helm and KUDO addons in the catalog deploy flow.

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ To deploy an addon, select a Project from your Workspace and select the **View C
 
 To customize an addon before deploying it, you can customize the helm chart values or KUDO parameters of a service before deploying it.
 
-![Deploy Platform Service](/ksphere/kommander/img/project-catalog-deploy.png)
+For helm-based addons, you can specify the helm chart values in a YAML editor:
```
Comment (samvantran) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md:31]:
```suggestion
For Helm-based addons, specify the chart values in a YAML editor:
```

File: pages/ksphere/kommander/latest/projects/addon-catalog/index.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ To deploy an addon, select a Project from your Workspace and select the **View C
...
+
+![Deploy Helm Platform Service](/ksphere/kommander/img/project-catalog-deploy.png)
+
+For KUDO-based addons, you can fill out the form fields corresponding to the specific properties that are supported by that addon's parameters schema:
```
Comment (samvantran) [inline pages/ksphere/kommander/latest/projects/addon-catalog/index.md:35]:
```suggestion
For KUDO-based addons, fill out the form fields supported by that addon's parameters schema:
```

**Comment by juliangieseke** [comment]
rebased on new `1.1.0-beta` folder/docs 👍

**Comment by juliangieseke** [comment]
@cmastr i think its ready for merge now

**Pull request merged**